Makefile

```
@@ -291,6 +293,11 @@ generate-manifests: $(CONTROLLER_GEN) ## Generate manifests e.g. CRD, RBAC etc.
 		output:crd:dir=$(CRD_ROOT) \
 		output:webhook:dir=$(WEBHOOK_ROOT) \
 		webhook
+	# Generate webhook manifests for supervisor mode separately.
+	$(CONTROLLER_GEN) \
+		paths=./internal/webhooks/vmoperator \
```

```suggestion
		paths=./internal/webhooks/vmware \
```
Let's call this vmware, analog to apis/vmware. I think vmoperator could be easily misunderstood

---

Fixed!